tests/FVTTests/environment.ts

```diff
@@ -9,6 +9,10 @@
  */
 export const USERNAME :string = process.env.ZOWE_USERNAME;
 export const PASSWORD :string = process.env.ZOWE_PASSWORD;
+export const SERVER_HOST :string = process.env.SERVER_HOST_NAME;
+export const SERVER_PORT :number = parseInt(process.env.SERVER_HTTPS_PORT);
 
-export const BASE_URL :string = `https://${process.env.SERVER_HOST_NAME}:${process.env.SERVER_HTTPS_PORT}`;
-export const BASE_URL_WITH_PATH :string =`${BASE_URL}/ui/v1/explorer-mvs`;
\ No newline at end of file
+export const BASE_URL :string = `https://${SERVER_HOST}:${SERVER_PORT}`;
+export const BASE_URL_WITH_PATH :string =`${BASE_URL}/ui/v1/explorer-mvs`;
+
+export const TEST_DATASET :string = `${USERNAME.toUpperCase()}.FVTTEST`;
```

Comment:
Do we have plan to further change this to a random name? When we use same name for testing, and 2 builds (for example staging branch and PR branch) are running at the same time, the creation and deletion of this test dataset will have potential to conflict. One build may delete the dataset while other build is testing on it.

Reply:
I've added a unique qualifier based on possix time stamp to the test dataset